models/issue_comment.go

```diff
@@ -75,23 +79,25 @@ const (
 
 // Comment represents a comment in commit and issue page.
 type Comment struct {
-	ID             int64 `xorm:"pk autoincr"`
-	Type           CommentType
-	PosterID       int64 `xorm:"INDEX"`
-	Poster         *User `xorm:"-"`
-	IssueID        int64 `xorm:"INDEX"`
-	LabelID        int64
-	Label          *Label `xorm:"-"`
-	OldMilestoneID int64
-	MilestoneID    int64
-	OldMilestone   *Milestone `xorm:"-"`
-	Milestone      *Milestone `xorm:"-"`
-	OldAssigneeID  int64
-	AssigneeID     int64
-	Assignee       *User `xorm:"-"`
-	OldAssignee    *User `xorm:"-"`
-	OldTitle       string
-	NewTitle       string
+	ID               int64 `xorm:"pk autoincr"`
+	Type             CommentType
+	PosterID         int64 `xorm:"INDEX"`
+	Poster           *User `xorm:"-"`
+	IssueID          int64 `xorm:"INDEX"`
+	LabelID          int64
+	Label            *Label `xorm:"-"`
+	OldMilestoneID   int64
+	MilestoneID      int64
+	OldMilestone     *Milestone `xorm:"-"`
+	Milestone        *Milestone `xorm:"-"`
+	OldAssigneeID    int64
+	AssigneeID       int64
+	Assignee         *User `xorm:"-"`
+	OldAssignee      *User `xorm:"-"`
+	OldTitle         string
+	NewTitle         string
+	DependentIssueID int64
```

Comment:
This means you can only have 1 dependency right?

Reply:
One per comment. (You can ofc have more than dependency per issue)

The ID here is used to get issue details (title, index) later when displaying the comment to be able to link to it.

Reply:
Why does Comments track dependencies? 😕

Reply:
They don't the ID is only used to display informations in a comment (title, index, link) when someone adds or removes a dependency.